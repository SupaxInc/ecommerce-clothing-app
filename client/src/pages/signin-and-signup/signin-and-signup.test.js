import React from "react";
import { shallow } from "enzyme";

import * as redux from 'react-redux';
import SignInAndSignUpPage from "./signin-and-signup";

describe('signin-and-signup page component', () => {
    let wrapper;

    beforeEach(() => {
        wrapper = shallow(<SignInAndSignUpPage />);
    });
    
    it('should render signin-and-signup page', () => {
        expect(wrapper).toMatchSnapshot();
    });
})