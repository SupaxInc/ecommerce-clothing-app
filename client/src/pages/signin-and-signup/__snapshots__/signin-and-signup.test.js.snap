// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`signin-and-signup page component should render signin-and-signup page 1`] = `ShallowWrapper {}`;
